package.json

```diff
@@ -53,6 +54,9 @@
     "react": ">= 16",
     "react-dom": ">= 16"
   },
+  "resolutions": {
+    "ansi-regex": "^3.0.1"
```

Comment:
why is this needed?

should be documented in the PR

Reply:
I explained in its [own commit](https://github.com/shakacode/react_on_rails/pull/1514/commits/a019f7e03a1dd5e8bfbc747549ae05ef60a7ff10):

> This is a temporary fix due to complexities of upgrading eslint and
> other relevant dependencies (namely prettier-eslint-cli) to newer
> versions.

Maybe I had to bring it to the PR body as well.

Since we had a kind of urgency in resolving high/critical vulnerabilities, I did a tradeoff and chose to use this approach for this particular package rather than spending hours or days to fix issues with upgrading `prettier-eslint-cli`, which required upgrading `eslint` and make modifications in our source code to address the breaking changes.

Reply:
Fixed ✅